### PR TITLE
Update docker-compose.yml

### PR DESCRIPTION
**Pull request by Angelo-gh3990**
Change this, and now it builds correctly. Don't know if this is what you want?

ref: https://github.com/filecoin-project/motion/issues/155

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -132,7 +132,7 @@ services:
       - motion-singularity-volume:/usr/src/app/storage
     depends_on:
       singularity_api:
-        condition: service_healthy
+        condition: service_completed_successfully
```
[the remark this answers is not in the document]
Comment (Angelo-gh3990) [reply docker-compose.yml:135]: No I agree, after looking into it more. It showed me for now where the issue was. Thanks

[the remark this answers is not in the document]
Comment (xinaxu) [reply docker-compose.yml:135]: There is already a /health endpoint to check for health in Singularity
https://github.com/data-preservation-programs/singularity/blob/7a3b2c61c4deb38aea50b6b8bd17a5ab75ff420d/api/api.go#L437C37-L437C37

[the remark this answers is not in the document]
Comment (xinaxu) [reply docker-compose.yml:135]: Oh that's only available with v0.5.0, below one should work though
http://127.0.0.1:9090/swagger/index.html

**Review by xinaxu — COMMENTED** [review]
see above

**Comment by gammazero** [comment]
So I think we can close this as it should be fixed by #164.

**Pull request closed without merging**